docs/services/sensors.md

```diff
@@ -0,0 +1,147 @@
+---
+title: "Sensors Service"
+linkTitle: "Sensors"
+weight: 70
+type: "docs"
+description: "The Sensors Service is a built-in service that provides a central interface to all of your robot's sensors."
+tags: ["sensor", "services"]
+icon: "/services/img/icons/sensor.svg"
+images: ["/services/img/icons/sensor.svg"]
+# SME: Cheuk
+---
+
+The Sensors Service is a built-in service that provides a central interface to all of your robot's [sensors](/components/sensor), regardless of the sensor model.
```

Comment:
```suggestion
The Sensors Service provides a central interface to all of your robot's [sensors](/components/sensor), regardless of the sensor model.
```
[opt] see above

Reply:
Built-in is supposed to mean you don't have to configure it yourself. Though why we do allow that through the UI is a good question...